lints/lint_mp_authority_key_identifier_correct.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * ZLint Copyright 2019 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/********************************************************************
+Section 5.2 - Forbidden and Required Practices
+CAs MUST NOT issue certificates that have:
+- incorrect extensions (e.g., SSL certificates that exclude SSL usage, or authority key IDs
+  that include both the key ID and the issuer’s issuer name and serial number); or
```

Comment:
tiny nit: cutting off the trailing "or" seems more natural for this context.

```suggestion
  that include both the key ID and the issuer’s issuer name and serial number);
```